MOD/Witcher/interface/portrait_properties/00_portrait_properties.txt

```diff
@@ -2673,10 +2673,11 @@
 				character = 155103
 				character = 155102
 				character = 9025
+				character = 65200
 			}
 		}
 	}
-	1 = { # Myszow�r
+	1 = { # Myszowór, aka Mousesack

```

Comment:
It's best to keep the file encoding as ANSI (Windows-1252) - which Github doesn't support � - since UTF-8 isn't well supported by CK2 engine.
With Notepad++ you can do Encoding / Convert to ANSI to fix it.

Reply:
Let me take a look.  I've been using both Visual Studio Code and Notepad++, and maybe something got mixed up.